app/controllers/api/v1/notification_tokens_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::NotificationTokensController < Api::BaseController
+
+  def create
+    @notification_token = current_user.notification_tokens.find_or_initialize_by(value: notification_token_params[:token])
+    if @notification_token.save
+      render json: {}, status: :created
```

Comment:
in order to avoid the save when the token already exists, we can do something like:

```suggestion
    @notification_token = current_user.notification_tokens.find_or_create_by(value: notification_token_params[:token])
    if @notification_token.valid?
      render json: {}, status: :created
```